sdk/policyinsights/Azure.ResourceManager.PolicyInsights/CHANGELOG.md

```diff
@@ -1,9 +1,16 @@
 # Release History
 
-## 1.1.0 (2022-11-01)
+## 1.1.0 (2023-02-16)
```

Comment:
Why this release is replacing an existing version?

Reply:
The service team has a PR to upgrade the API version in Nov, this version was set at that time. But unfortunately, we didn't release it after the PR is merged.